guides/common/modules/snip_host-names-for-http-proxy.adoc

```diff
@@ -9,6 +9,7 @@ ifdef::satellite[]
 | cert.console.redhat.com (if using Red{nbsp}Hat Insights) | 443 | HTTPS
 | api.access.redhat.com (if using Red{nbsp}Hat Insights) | 443 | HTTPS
 | cert-api.access.redhat.com (if using Red{nbsp}Hat Insights) | 443 | HTTPS
+| console.redhat.com (if using Red{nbsp}Hat Insights) | 443 | HTTPS
```

Comment:
Do we need to add `connect.cloud.redhat.com` here too?

Reply:
@ShimShtein done